src/polysynth.js

```diff
@@ -389,6 +387,7 @@ class PolySynth {
         this.notes[n].dispose();
         delete this.notes[n];
       }
+      this._newest = 0;
```

Comment:
I'm curious what test that you added to uncover this bug (I can take a look at your other PR to find out! But in hindsight / next time, it would've been useful to include the unit tests and bug fixes in the same PR)

Reply:
I do not think a particular test uncovered this. While testing the function to include a case that if a note is not provided, it should reset all the notes. It seemed logical to reset the newest value as all the notes are being erased

Reply:
Actually, I think we should also reset the oldest value also right?